### ISSUE TITLE
PowerPoint: NVDA no longer announces bullets when creating and reading presentation slides

### ISSUE THREAD

**Issue by josephsl** [nvaccess/nvda#17309]
### Steps to reproduce:
Prerequisite: PowerPoint (either 365 or Office 2021) must be installed.

1. Create a presentation with at least one content slide (no center title slide).
2. After filling in the slide title, move to object portion.
3. Fill out the "object" i.e. content portion of the slide. By default, a bullet point should be present.
4. After writing one line, press Enter to add another bulleted list item in the slide content.
5. Return to the slide view and press Tab to review title and content (one should press Tab twice to move to slide object/content).

### Actual behavior:
NVDA does not announce bullet points when composing and reading slides.

### Expected behavior:
NVDA reads bullet points when composing and reading slides.

### NVDA logs, crash dumps and other attachments:
None

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
alpha-34344,64ba6c69

#### Windows version:
Windows 11 24H2 dev (build 26120.2130)

#### Name and version of other software in use when reproducing the issue:
PowerPoint 365 version 16.0.18025.20140 (64-bit)

#### Other information about your system:
Used as a development workstation and PowerPoint is used as part of a job

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Not reproducible in 2024.4 RC and earlier (also not reproducible in earlier alpha builds but can't pinpoint to the exact build/commit at the moment)

#### If NVDA add-ons are disabled, is your problem still occurring?
Yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Not applicable

Thanks.

**Comment by RajathKP** [comment]
I would like to work on this. Please assign

**Comment by seanbudd** [comment]
Thanks @RajathKP - let me know if you need any questions when getting started

**Comment by RajathKP** [comment]
Sure

**Comment by Qchristensen** [comment]
@RajathKP Have you had any luck looking at this issue?

Meanwhile, I had a user report this and in investigating, in addition to the original comments, I found:

* If you press shift+enter to create a new line without a new bullet, as you navigate through the slide, NVDA reports a bullet on THIS line but not the others.
* When running the slideshow (f5), bullets are announced correctly.
* Narrator announces bullets correctly when navigating while editing 

Using nvda_snapshot_alpha-55570,bcd5eb66.exe
Windows 11 (64-bit) Version: 25H2, Build: 26200.8246 
Office 365 (64-bit) Version: 16.0.19822.20182

**Comment by Bibeksharma5** [comment]
> [@RajathKP](https://github.com/RajathKP) Have you had any luck looking at this issue?
> 
> Meanwhile, I had a user report this and in investigating, in addition to the original comments, I found:
> 
> * If you press shift+enter to create a new line without a new bullet, as you navigate through the slide, NVDA reports a bullet on THIS line but not the others.
> * When running the slideshow (f5), bullets are announced correctly.
> * Narrator announces bullets correctly when navigating while editing
> 
> Using nvda_snapshot_alpha-55570,bcd5eb66.exe Windows 11 (64-bit) Version: 25H2, Build: 26200.8246 Office 365 (64-bit) Version: 16.0.19822.20182

Yes, you're right. Generally, if I need to write the line without bullet, I press Shift+Enter. That time, no bullet got created. But, NVDA announces "Bullet" followed by the line. And if I want to create a next line with bullet, I press enter. That time, bullet is created. But, NVDA doesn't say "bullet". This is just reversed. So, please fix this issue.
